deeppavlov/models/embedders/elmo_embedder.py

```diff
@@ -12,18 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from overrides import overrides
-from typing import List, Union, Iterator
-import tensorflow as tf
+import sys
+from typing import Iterator, List, Union
 
 import numpy as np
+import tensorflow as tf
 import tensorflow_hub as hub
-
 from deeppavlov.core.commands.utils import expand_path
-from deeppavlov.core.common.registry import register
-from deeppavlov.core.models.component import Component
 from deeppavlov.core.common.log import get_logger
+from deeppavlov.core.common.registry import register
 from deeppavlov.core.data.utils import zero_pad
+from deeppavlov.core.models.component import Component
+from overrides import overrides
```

Comment:
This import should be placed in the middle bunch with numpy and tensorflow to comply with [pep8](https://www.python.org/dev/peps/pep-0008/?#imports)

Reply:
it is updated in 9efeba2 commit.